### PR TITLE
Add portfolio projects

### PR DESCRIPTION
**Pull request by allenkays**
In this milestone I:

- Added my recent works to the portfolio.
- Added button interactions.
- Added about me section and my stack.
- Updated the styles.css  file to style the portions added to the index page.
- Installed the node modules for linters checks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,7 +43,196 @@ <h2 class="sub-title">I'm a Software Developer</h2>
                         </li>
                     </ul>
                 </article>
-            </main>
-        </section>
+            </section>
+            <section class="portfolio">
```
Comment (Shisui6) [inline index.html:47]: According to the project requirements, you are to use CSS grid to style the main container of the works section. Currently your main container is not using the CSS grid property. Please update your works section to contain this requirement🙏

File: index.html [hunk 2]
```diff
@@ -43,7 +43,196 @@ <h2 class="sub-title">I'm a Software Developer</h2>
...
+                        <div class="grid-item"></div>
+                        <div class="grid-item">
+                            <button class="b-button" type="button">See project</button>
+                        </div>
+                        <div class="grid-item"></div>
+                    </div>
+                </article>
+            <section class="about">
+                <article class="about-me">
+                    <h1 class="main-title">About me</h1>
```
Comment (Shisui6) [inline index.html]: Kindly update this element to use another `<hx>` tag apart from `<h1>`. It is considered best practice for a webpage to have only one h1 tag which represents the main title.🙏

File: index.html [hunk 3]
```diff
@@ -43,7 +43,196 @@ <h2 class="sub-title">I'm a Software Developer</h2>
...
+                            <img src="assets/about/Rectangle_57.png" alt="shape">
+                            <h3>Skills</h3>
+                        </span>
+                        <ul class="lang-frameworks">
+                            <li class="stack3">Database Management</li>
+                            <li class="stack3">Version Control</li>
+                            <li class="stack3">CLI</li>
+                            <li class="stack3">API Design</li>
+                            <li class="stack3">Web Development</li>
+                        </ul>
```
Comment (Shisui6) [inline index.html]: Kindly update this section to use a `<ul>` tag to wrap all the technology categories, as it is one of the project requirements🙏.

image

Also, ensure that each of the technology categories is also wrapped in `<li>` tags. 🫡

File: index.html [hunk 4]
```diff
@@ -43,7 +43,196 @@ <h2 class="sub-title">I'm a Software Developer</h2>
...
+                        <div class="grid-item"></div>
+                    </div>
+                </article>
+            <section class="about">
+                <article class="about-me">
+                    <h1 class="main-title">About me</h1>
+                    <p class="support-text">
+                        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+                    <button class="resume" type="button">Get my resume</button>
```
Comment (Shisui6) [inline index.html]: - Nice job styling your buttons when hovered. However the colours shown when the button is hovered over is not in line with the figma design. Please try to stick to the colors used in the figma design as much as possible. 
- Also, please apply styles to your buttons when they are pressed in accordance with the figma design. You can use the `:active` pseudo selector to achieve this

image

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @allenkays, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (allenkays) [reply index.html:47]: I've made the changes in css files to apply grid layout to this section

File: index.html (hunk 2 above)
Comment (allenkays) [reply index.html]: I have changed this tag to <h2>. Thanks for the best practice reminder.

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**